public/app/screens/main/navigation/navigation.jsx

```diff
@@ -27,7 +27,7 @@ var Navigation = React.createClass({
     }.bind(this));
 
     return (
-      <div className={'cc_navigation ' + this.props.orientation}>
+      <div className={'cc_navigation icon-bar two-up' + ' ' + this.props.orientation}>
```

Comment:
When showing more than two nav items, does `two-up` need to be replaced with `three-up`?